### PR TITLE
Add user who submitted a page for moderation to the home screen table in `WorkflowPagesToModeratePanel`

### PR DESCRIPTION
**Pull request by Tijani-Dia**
This PR adds the user who submitted a page for moderation in the `WorkflowPagesToModeratePanel`.

Screenshot from 2022-01-11 10-46-50

* Do the tests still pass?Yes
* Does the code comply with the style guide? Yes
* For Python changes: Have you added tests to cover the new/fixed behaviour?
* For front-end changes: Did you test on all of [Wagtail’s supported environments]? No
    * **Please list the exact browser and operating system versions you tested**. Firefox
    * **Please list which assistive technologies you tested**. None
* For new features: Has the documentation been updated accordingly? No

### CODE REVIEW

File: wagtail/admin/templates/wagtailadmin/home/workflow_pages_to_moderate.html [hunk 1]
```diff
@@ -63,6 +65,9 @@ <h2 class="title-wrapper">{% trans 'Awaiting your review' %}</h2>
                                     </span>
                                 {% endfor %}
                             </td>
+                            <td valign="top">
+                                {{ revision.user }}
```
Comment (zerolab) [inline wagtail/admin/templates/wagtailadmin/home/workflow_pages_to_moderate.html:69]: can you that this doesn't add an n+1 query?
I have a suspicion `WorkflowPagesToModeratePanel` needs `page_revision__user` added to the queryser `select_related()` declaration

Comment (Tijani-Dia) [reply wagtail/admin/templates/wagtailadmin/home/workflow_pages_to_moderate.html:69]: You are right!

Comment (zerolab) [inline wagtail/admin/templates/wagtailadmin/home/workflow_pages_to_moderate.html:69]: suggestion:
```suggestion
                                {{ revision.user|user_display_name }}
```

for consistency with other areas where show the user.

Other than that LGTM

**Review by zerolab — CHANGES_REQUESTED** [review]

File: wagtail/admin/templates/wagtailadmin/home/workflow_pages_to_moderate.html (hunk 1 above)
Comment (zerolab) [reply wagtail/admin/templates/wagtailadmin/home/workflow_pages_to_moderate.html:69]: discussed in Slack, will address during merge

**Comment by zerolab** [comment]
Cleaned up and merged in f3f35f6. Thank you @Tijani-Dia

**Pull request closed without merging**